testutils.py

```diff
@@ -51,3 +51,16 @@ def if_rollback():
     if "ROLLBACK TARGET" in out:
         return True
     return False
+
+def get_fedora_release():
+    "Finds the fedora version in str"
+    ver = ""
+    with open("/etc/os-release") as fobj:
```

Comment:
@kushaldas Just cursious: we can have `/etc/fedora-release` instead since we are targeting only `fedora` distribution? This could make the method few liner :-).